### DISCUSSION TITLE
Teleporting Unconscious Characters

### DISCUSSION THREAD

**Discussion by thesuntsar** [Arx-Game/arxcode#393] [Ideas]
Would it be possible to code a command that works solely when an event log is running for event GMs to be able to teleport unconscious PCs out of plot rooms to either Saving Grace Hospital or the House of Solace? Sometimes scenes end at weird hours so this would just be to make it a little more efficient to get them back into the flow of RP and so people don't feel that ping of guilt bugging someone to teleport them.

**Comment by TehomCD** [comment]
Yeah, this is a good idea. Short term this could be a few rooms that have been tagged as hospital locations and let you pick one of those
